core/src/server/component.cpp

```diff
@@ -128,6 +128,9 @@ additionalProperties: false
                     cert:
                         type: string
                         description: path to TLS certificate
+                    cert-chain:
```

Comment:
Please, also add the description to https://github.com/userver-framework/userver/blob/bb37e2121cc43ea9317d089435c6ac8e5c41e51b/core/include/userver/server/component.hpp#L66

Reply:
added